utils/scraping.py

```diff
@@ -0,0 +1,53 @@
+import os
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+SELENIUM_DRIVER = os.environ.get("SELENIUM_DRIVER")
+driver = webdriver.Firefox(executable_path=SELENIUM_DRIVER)
+
+
+def scrapeQuotes(url) -> list:
```

Comment:
Follow snake case for function and variable names. Update all occurrences.

Reply:
Please add a reply comment mentioning you resolved this issue so that I can know about the changes.